infra/stack.json

```diff
@@ -0,0 +1,415 @@
+            "NoEcho": "true",
+            "Description": "Secret key for Play application",
+            "Type": "String",
+            "MinLength": "12",
+            "MaxLength": "64"
+        }
+    },
+    "Resources": {
+        "vpc": {
+           "Type" : "AWS::EC2::VPC",
+           "Properties" : {
+              "CidrBlock" : "10.192.0.0/16"
+           }
+        },
+        "ig": {
+              "Type" : "AWS::EC2::InternetGateway",
+              "Properties" : {}
+        },
+        "pubsubnet1" : {
+           "Type" : "AWS::EC2::Subnet",
+           "Properties" : {
+              "CidrBlock" : "10.192.10.0/24",
+              "VpcId" : { "Ref" : "vpc" },
+              "AvailabilityZone": "us-west-2a",
+              "MapPublicIpOnLaunch": true
+           }
+        },
+        "pubsubnet2" : {
+           "Type" : "AWS::EC2::Subnet",
+           "Properties" : {
+              "CidrBlock" : "10.192.11.0/24",
+              "VpcId" : { "Ref" : "vpc" },
+              "AvailabilityZone": "us-west-2b",
+              "MapPublicIpOnLaunch": true
+           }
+        },
+        "privsubnet1" : {
+           "Type" : "AWS::EC2::Subnet",
+           "Properties" : {
+              "CidrBlock" : "10.192.20.0/24",
+              "VpcId" : { "Ref" : "vpc" },
+              "AvailabilityZone": "us-west-2a",
+              "MapPublicIpOnLaunch": false
+           }
+        },
+        "privsubnet2" : {
+           "Type" : "AWS::EC2::Subnet",
+           "Properties" : {
+              "CidrBlock" : "10.192.21.0/24",
+              "VpcId" : { "Ref" : "vpc" },
+              "AvailabilityZone": "us-west-2b",
+              "MapPublicIpOnLaunch": false
+           }
+        },
+        "attachgateway" : {
+           "Type" : "AWS::EC2::VPCGatewayAttachment",
+           "Properties" : {
+              "VpcId" : { "Ref" : "vpc" },
+              "InternetGatewayId" : { "Ref" : "ig" }
+            }
+        },
+        "natgateway1ip" : {
+           "Type" : "AWS::EC2::EIP",
+           "DependsOn": ["attachgateway"],
+           "Properties": {
+                   "Domain": {"Ref": "vpc"}
+           }
+        },
+        "natgateway2ip" : {
+           "Type" : "AWS::EC2::EIP",
+           "DependsOn": ["attachgateway"],
+           "Properties": {
+                   "Domain": {"Ref": "vpc"}
+           }
+        },
+        "natgateway1": {
+                "Type": "AWS::EC2::NatGateway",
+                "Properties": {
+                        "AllocationId": {"Fn::GetAtt": ["natgateway1ip", "AllocationId"]},
+                        "SubnetId": {"Ref": "pubsubnet1"}
+                }
+        },
+        "natgateway2": {
+                "Type": "AWS::EC2::NatGateway",
+                "Properties": {
+                        "AllocationId": {"Fn::GetAtt": ["natgateway2ip", "AllocationId"]},
+                        "SubnetId": {"Ref": "pubsubnet2"}
+                }
+        },
+        "routetable" : {
```

Comment:
**Naming Suggestion** (non-blocking): The route table connects to internet gateway. Shall we call this `igroutetable` or `pubroutetable`?

Reply:
Done. :)